ProcessLib/BoundaryCondition/ConstraintDirichletBoundaryCondition.cpp

```diff
@@ -139,8 +139,10 @@ void ConstraintDirichletBoundaryCondition::getEssentialBCValues(
     std::vector<std::pair<GlobalIndexType, double>> tmp_bc_values;
 
     auto isFlux = [&](const std::size_t element_id) {
-        return _lower ? _flux_values[element_id] < _constraint_threshold
-                      : _flux_values[element_id] > _constraint_threshold;
+        return _lower ? _constraint_threshold - _flux_values[element_id] >=
+                            std::numeric_limits<double>::epsilon()
+                      : _flux_values[element_id] - _constraint_threshold >=
+                            std::numeric_limits<double>::epsilon();
```

Comment:
Not sure, if this is a good idea...
I see how it solves the current ctest problem, though.
If this is what the boundary condition is required to do, OK, but I have a feeling, that this is not right way.

Documentation update might be needed for the changed behaviour.

Reply:
The implementation requires that the user is familiar with correct (physical) setting the boundary condition.

Updated documentation.